### DISCUSSION TITLE
Lower the big instance cd from 7 days to 3 days

### DISCUSSION THREAD

**Discussion by doppiap** [chromiecraft/chromiecraft#3568] [Suggestions]
to combat the monotony of the server after doing pg, alt, archivement, jobs, farming, people are going to other servers to continue playing (in my guild 1 month ago in the afternoon there were at least 20 people, today 3). To overcome this problem, a possible solution could be to reduce the cd of the big instances(MC and BWL for now), to lower the cd from 7 days to 3 days. So you can entertain other months to farm sets and items for main pg and alt. Beyond that, doing so many raids would help the server economy. People will need more consumables to raid, so people would farm more to make more money to be able to buy those consumables.

**Comment by Santaria94** [comment]
It seems to me that if someone wants more raids, then he just creates additional alt characters. I dont think that the server needs such deviations from Blizzlike

**Reply by doppiap** [reply]
lol, I'll tell you a secret, this server is not blizzlike.
Instances do not cross factions, guilds do not cross factions, do not talk to other factions (if you speak they do not understand you), world drops are not as rich as in cromiecraft, instances like MC and BWL are 40 and not from 20, the talents and mechanics we have are from 3.3.5 and not from 1.xx, continue?
world of warcraft is an MMORPG, the real experience would be to use only one character, you are that character, you shouldn't have others. if to have fun you have to do N pg, something is wrong.

**Comment by KaneoHunter** [comment]
Please no I already maintain 4 characters on a 7 day cycle, I don't wanna do 4 raids every 3 days.

**Comment by Annamaria-CC** [comment]
Thanks for your suggestion.

We do not think reducing the raid lockout is a benefit to the server. Reducing it would mean people would obtain their items much quicker, and therefore will be more bored more quicker. We therefore deny your suggestion.

Locking this discussion.
